src/core/xref.js

```diff
@@ -139,9 +139,14 @@ function getRequestEntry(elem) {
   let dataciteElem = elem.closest("[data-cite]");
   while (dataciteElem) {
     const cite = dataciteElem.dataset.cite.toLowerCase().replace(/[!?]/g, "");
-    const cites = cite.split(/\s+/).filter(s => s);
-    if (cites.length) {
-      specs.push(cites.sort());
+    const cites = cite.split(/\s+/);
+    // if we already have a spec in previous level of fallback chain, skip it
```

Comment:
By "previous" it means "higher" right?

Reply:
Lower level actually - as we're traversing up towards root. Want me to change the comment?

> Can this be a separate function

Will do in another PR.

Reply:
> Lower level actually

But that level has higher priority, right?

>Will do in another PR.

I would prefer doing it here if no blocker 👀

Reply:
Lower level is higher priority - correct.

> I would prefer doing it here if no blocker :eyes:

:+1:

Reply:
It was initially a bit ambiguous for me what "previous" mean there. Maybe just "subtree"?